.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Context
```

Comment:
Why does this one have examples and the other one doesn't?

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2416/commits/47649204fd963ba209a30fd76b240efa0dab2616